src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.tsx

```diff
@@ -1,7 +1,7 @@
-import { BoxProps } from '@stacks/ui';
 import { forwardRefWithAs } from '@stacks/ui-core';
```

Comment:
I don't know / understand what this `forwardRefWithAs` does. Maybe we can remove/ migrate it in a separate task?

Reply:
> I don't know / understand what this `forwardRefWithAs` does. Maybe we can remove/ migrate it in a separate task? 

It allows the 'as' prop. I have removed it in all the files listed for part 3. We should remove as we go.